roles/splunk_common/tasks/check_mgmt_mode_status.yml

```diff
@@ -7,14 +7,19 @@
 - name: Check if listening on SVC Port {{ splunk.svc_port }}
```

Comment:
I'd suggest to get rid of this task. Since this will add more dependency and we should keep splunk-ansible OS-agnostic

Reply:
done, removed netstat check